backend/server.js

```diff
@@ -64,6 +64,23 @@ let register_email_users = {};
 const FB = require('fb');
 FB.options({version: 'v2.8'});
 
+// wrapper for the FB API to get access token each time
+const FBReq = (req, cb) => {
+  FB.api('oauth/access_token', {
+    client_id: process.env.ITERATE_FB_APP_ID,
+    client_secret: process.env.ITERATE_FB_APP_SECRET,
+    grant_type: 'client_credentials'
+  }, res => {
+    if (!res || res.error) {
+      console.log(!res ? 'error occurred during the token request' : res.error);
+      return;
+    }
+    const { access_token } = res;
+    FB.setAccessToken(access_token);
```

Comment:
Also semantically would be better to use `console.error`, not `console.log`. In any case there's a ticket to use `morgan`, apparently someone is working on that.